cost-analyzer/values.yaml

```diff
@@ -706,12 +706,24 @@ awsstore:
   useAwsStore: false
   createServiceAccount: false
 
+federatedETL:
```

Comment:
Can we add a quick summary/comments for these or link to docs?

Reply:
Yep, added explanatory comments and will link the docs shortly.